src/components/withProps.test.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { mount, shallow } from 'enzyme';
```

Comment:
`import { ... }` comes after `import Package`, i.e. please move this down 1 line.